packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { AmountInput, TransferAccountSelector } from '@deriv/components';
+import { getDecimalPlaces } from '@deriv/shared';
+import { useStore, observer } from '@deriv/stores';
+import { localize, Localize } from '@deriv/translations';
+import { transfer_accounts } from './mock_accounts/mock_accounts';
+import type { TAccount } from './types';
+import './wallet-transfer.scss';
+
+const Divider = () => <div className='wallet-transfer__divider' />;
+
+const WalletTransfer = observer(() => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
```

Comment:
```suggestion
    import { isMobile } from '@deriv/shared';

```

Reply:
@mahdiyeh-deriv as far i know we decided to use is_mobile from store. you can check this [link](https://deriv-group.slack.com/archives/C3AHMSM3P/p1683037206653169)